docs/nidcpower/functions.rst

```diff
@@ -721,14 +721,22 @@ nidcpower.Session methods
 
 .. py:method:: measure_multiple()
 
-    Returns arrays of the measured voltage and current values on the
-    specified output channel(s). Each call to this method blocks other
-    method calls until the measurements are returned from the device. The
-    order of the measurements returned in the array corresponds to the order
-    on the specified output channel(s).
+    Returns an list of named tuples (Measurement) that were
```

Comment:
a list

Reply:
This documentation seems wrong.
measure_multiple doesn't retrieve stuff from a buffer. It actually performs the measurements.
The note I think applies to fetch. Copy / paste error?

Reply:
Updated